### DISCUSSION TITLE
Weather widget stopped working

### DISCUSSION THREAD

**Discussion by MMaicki** [valinet/ExplorerPatcher#1304] [General]
![image](https://user-images.githubusercontent.com/46030081/231413347-4d91acfd-6d42-4c69-9cf0-d654a67e6da6.png)
![image](https://user-images.githubusercontent.com/46030081/231414724-d358f3d8-de8b-498b-a067-3f26f115ddb7.png)

Weather widget isn't working anymore after the update.

Widget is flickering through Google websites like accepting terms of service and it fails to load the data.

**Comment by JakeWerecat** [comment]
Duplicate of: https://github.com/valinet/ExplorerPatcher/discussions/1301

**Comment by jw-mn** [comment]
Got the same issue. Rolled back to 22621.1344.53.1. Uninstall and reinstall the older version. This fixed it for me.

Check here: https://github.com/valinet/ExplorerPatcher/issues/1310

**Reply by MMaicki** [reply]
Rollback helped, thanks.

**Reply by sergyum** [reply]
This helped, need to manually resize after each pc reboot to see properly, but it works Thanks!
